registry/default/blocks/post-page/mock-data.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from "react";
+import { Ellipsis } from "lucide-react";
+
+import { ContentCard } from "@/registry/default/blocks/content-card";
+import {
+  DropdownMenuItem,
+  DropdownMenuSeparator,
+} from "@/registry/default/ui/dropdown-menu";
+
+import { ReplyCard } from "../reply-card";
+
+const MENU = {
+  trigger: (
+    <Ellipsis
+      size={20}
+      className="text-primary cursor-pointer hover:text-muted-foreground"
+    />
+  ),
+  items: [
+    <DropdownMenuItem key="flag"> Flag </DropdownMenuItem>,
+    <DropdownMenuItem key="delete"> Delete </DropdownMenuItem>,
+    <DropdownMenuSeparator key="separator" />,
+    <DropdownMenuItem key="edit"> Edit </DropdownMenuItem>,
```

Comment:
why ? 
1. `DropdownMenuItem` should be inside the content card
2. items: ["flag", "delete', "edit"]
3. if you need to introduce separator then the items should be defined as 
`items: { label: string; separator: boolean}`
  then manage the separate logic inside the content card

Reply:
@kunstefix the I didnt address about passing items as ReactNode. I am asking why expose `DropdownMenuItem` outside ?